Streetcode/Streetcode.WebApi/Program.cs

```diff
@@ -14,14 +16,15 @@
 builder.Services.ConfigureBlob(builder);
 builder.Services.ConfigurePayment(builder);
 builder.Services.ConfigureInstagram(builder);
-
+builder.Services.ConfigureSerilog(builder);
 var app = builder.Build();
 
 if (app.Environment.EnvironmentName == "Local")
 {
     app.UseSwagger();
     app.UseSwaggerUI(c => c.SwaggerEndpoint("/swagger/v1/swagger.json", "WebAPIv5 v1"));
     await app.ApplyMigrations();
+    /* await app.SeedDataAsync(); */
```

Comment:
??

Reply:
this line is OK, we use it very often, because of need to delete Database time to time, so when we do it, we uncomment it